horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/user/KeycloakUserBackend.java

```diff
@@ -149,7 +149,7 @@ private static UserRepresentation convertUserRepresentation(UserService.NewUser
 
         CredentialRepresentation credentials = new CredentialRepresentation();
         credentials.setType(CredentialRepresentation.PASSWORD);
-        credentials.setTemporary(true);
+//        credentials.setTemporary(true);
```

Comment:
if this is no longer required, please remove

Reply:
removed. it was required, but I was now able to work around it,